CONTRIBUTING.md

```diff
@@ -10,18 +10,8 @@ Our goal is to make contributing to Aptos Core easy and transparent. See [Aptos
 ## Aptos Core
 
 To contribute to the Aptos Core implementation, first start with the proper development copy.
-
-To get the development installation with all the necessary dependencies for linting, testing, and building the documentation, run the following:
-```bash
-# fork https://github.com/aptos-labs/aptos-core.git via the github UI
-# repace <your-github> below to clone your (origin) remote fork
-git clone https://github.com/<your-github>/aptos-core.git
-cd aptos-core
-git remote add upstream https://github.com/aptos-labs/aptos-core.git
-./scripts/dev_setup.sh
-cargo build
-cargo test
-```
+You may want to use the GitHub interface to fork [aptos-core](https://github.com/aptos-labs/aptos-core) and check out your fork.
+For development environment setup and first build, see [Building Aptos From Source](developer-docs-site/docs/guides/building-from-source.md)
```

Comment:
So I am thinking we should probably make this an absolute link to the dev site:
* This is syntactically correct
* It likely takes the user into a self-hosted mode of the docs site that doesn't necessarily represent the experience we want them to have

Reply:
Done (as long as I got the formula right for what the aptos.dev link _will be_, since this new page isn't there yet.